docs/cloud-adoption/operations/operational-fitness-review.md

```diff
@@ -0,0 +1,127 @@
+
+This process is iterative, and ideally your enterprise should have a team dedicated to owning it. This team should meet regularly to review existing remediation projects, kick off the fundamentals review of new workloads, and track the enterprise's overall scorecard. The team should have the authority to ensure accountability for remediation teams that are behind schedule or fail to meet metrics.
+
+## Operational fitness review team structure
+
+The operational fitness review team is composed of three roles:
+
+1. **Business owner** : this role provides knowledge of the business to identify and prioritize each "mission critical" business operation. This role also compares the mitigation cost to the business impact and drive the final decision on remediation.
+
+2. **Business advocate** : this role requires deep knowledge of the technology associated with each business operation. This role is responsible for breaking down business operations into discreet parts and mapping those parts to on-premises and cloud services and infrastructure.
+
+3. **Engineering owner** : the role or roles responsible for implementation of the services associated with the business operation. These individuals may participate in the design, implementation, and deployment of any solutions for solving non-functional requirement issues uncovered by the operational fitness review team.
+
+4. **Service owner** : the role or roles responsible for operating the business's applications and services. These individuals collect logging and usage data for these applications and services, and this logging and usage data is used both to identify issues and verify fixes once deployed.
+
+## Fundamentals review example
+
+Contoso is an online retailer that has recently added the sales of event tickets to their business. Contoso creates a new [e-commerce front-end on Azure](../../example-scenario/apps/ecommerce-scenario.md) to perform the following functions:
+
+- Host the tickets sales web site front end. Users sign in to view concert details, order tickets, check the status of ordered tickets, and view and show tickets to gain entry to the concert venue. Images and other static content are served to the user through a content delivery network (CDN).
+- Payments are made using a third-party payment-processing system.
+- Users input concert reviews after the event has occurred, and machine learning is applied to the reviews to determine concert-goer sentiment which is provided to the concert promoter.
+
+Contoso has recently sold tickets for several popular concerts but has received complaints from customers that it's difficult to view concert details and order tickets once they go on sale. Several customers have also complained that they are randomly logged out of the site. Contoso has analyzed the performance of the application code and determined that there are no bugs in any feature that was cause the performance degradations or issues with authentication.
```

Comment:
> it's difficult to view concert details

I thought this referred to the UI design or something, but I think it means the application is timing out or otherwise unavailable?